File: atomichunter-rest/src/main/java/net/vniia/skittles/dto/StaffUnitDto.java
package net.vniia.skittles.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import net.vniia.skittles.enums.StaffUnitStatus;

import java.time.Instant;
import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class StaffUnitDto {
    private UUID id;
    private String positionId;
    private PositionDto position;
    private UUID employeeId;
    private StaffUnitStatus status;
    private Instant closeTime;
}

File: atomichunter-web/src/app/dto/StaffUnitDto.ts
import {Position} from "./Position";
import {Employee} from "./Employee";
import {StatusEnum} from "./status-enum";

export class StaffUnitDto {
    id: string;
    positionId: string;
    employeeId: string;
    employee: Employee;
    status: StatusEnum;
    position: Position;
    closeTime: Date;
}
